### ISSUE TITLE
Missing Polyplopia info in some frames. (Trac #2411)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2403]
Some frames after MPHitFilter are missing Polyplopia info.
The simprod-scripts for clsim also need to give an option to remove background-only events in which the primary particle has been pruned (because of lack of MCPES).

Migrated from https://code.icecube.wisc.edu/ticket/2411
```json
{
    "status": "closed", 
    "changetime": "2020-02-19T21:31:01", 
    "description": "Some frames after MPHitFilter are missing Polyplopia info.\nThe simprod-scripts for clsim also need to give an option to remove background-only events in which the primary particle has been pruned (because of lack of MCPES).", 
    "reporter": "juancarlos", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1582147861063941", 
    "component": "combo core", 
    "summary": "Missing Polyplopia info in some frames.", 
    "priority": "major", 
    "keywords": "", 
    "time": "2020-02-19T21:25:08", 
    "milestone": "", 
    "owner": "juancarlos", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20200219T21:28:00: juancarlos commented:

> This appears to be a problem with an IF statement that turns off prunning when the parent particle of an MCPE is undetermined because npe > 1. 
> Solution is to move the code that adds the polyplopia info outside the prunning condition statement.

**Comment by icecube-trac** [comment]
Trac update at 20200219T21:31:01: 
* juancarlos changed resolution from "" to "fixed"
* juancarlos commented:

> In changeset 179000:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="179000"
> Fixes #2403 and adds a couple of objects to keep list for L1
> ```


* juancarlos changed status from "new" to "closed"
